Libraries/Text/TextInput/Singleline/RCTSinglelineTextInputView.m

```diff
@@ -28,6 +28,8 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
 
     _backedTextInputView = [[RCTUITextField alloc] initWithFrame:self.bounds];
     _backedTextInputView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
+    _backedTextInputView.cell.scrollable = YES;
```

Comment:
should this be a prop?

Reply:
I think it should be okay if that's the default case / something we don't want to expose.